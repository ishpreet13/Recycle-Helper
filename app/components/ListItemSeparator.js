import React from "react";
import { View, StyleSheet } from "react-native";

import colors from "../config/colors";

function ListItemSeparator() {
  return <View style={styles.separator}></View>;
}

const styles = StyleSheet.create({
  separator: {
    width: "100%",
    height: 2,
    backgroundColor: "#d3d3d3",
  },
});

export default ListItemSeparator;
