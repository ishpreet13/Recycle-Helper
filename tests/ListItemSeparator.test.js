import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import ListItemSeparator from "../app/components/ListItemSeparator";

describe("Testing ListItemSeparator Component ", () => {
  const wrapper = renderer.create(<ListItemSeparator />);
  it("should render correctly", () => {
    expect(wrapper.toJSON()).toBeTruthy();
  });
});

describe("<ListItemSeparator />", () => {
  let tree;
  it("should render correctly", async () => {
    tree = renderer.create(<ListItemSeparator />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
