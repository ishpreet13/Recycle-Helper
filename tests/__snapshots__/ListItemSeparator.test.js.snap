// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItemSeparator /> should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#d3d3d3",
      "height": 2,
      "width": "100%",
    }
  }
/>
`;
